docs/how-to-guides/block-tutorial/block-controls-toolbar-and-sidebar.md

```diff
@@ -192,17 +182,100 @@ registerBlockType( 'gutenberg-examples/example-04-controls-esnext', {
 
 Note that `BlockControls` is only visible when the block is currently selected and in visual editing mode. `BlockControls` are not shown when editing a block in HTML editing mode.
 
-## Inspector
+## Sidebar Inspector
```

Comment:
```suggestion
## Settings Sidebar
```

Per the [Glossary](https://developer.wordpress.org/block-editor/handbook/glossary/) the sidebar is referred to as the Settings Sidebar so we should be consistent with the name throughout.